lib/decompress/zstd_decompress_internal.h

```diff
@@ -95,6 +95,9 @@ typedef enum {
     ZSTD_use_once = 1            /* Use the dictionary once and set to ZSTD_dont_use */
 } ZSTD_dictUses_e;
 
+#define ZSTD_OVERSIZED_MAXDURATION 128
```

Comment:
Should we just use the same macros as on the compression side? Should we find a place to put them where they can be shared?

Otherwise we should disambiguate the names so this clearly refers to dctx oversized calculation and the existing ones should be renamed to scope them to cctx management.